applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -244,7 +245,7 @@ static void syncable_cb(struct bt_audio_broadcast_sink *sink, bool encrypted)
 {
 	int ret;
 
-	if (init_routine_completed) {
+	if (streams[0].ep->status.state == BT_AUDIO_EP_STATE_STREAMING || !playing_state) {
```

Comment:
Would like to make sure if my understand is correct or not:
Since the syncable_cb will be triggered frequently when headset gets a sync_info packet, so we check playing_state, if it is not in playing then just return.

Reply:
Correct